staging/src/k8s.io/csi-api/pkg/crd/testdata/csidriver.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
```

Comment:
Please omit this.

Reply:
Same here, this is the default serialization